libnetwork_test.go

```diff
@@ -97,7 +97,7 @@ func getPortMapping() []types.PortBinding {
 }
 
 func TestNull(t *testing.T) {
-	network, err := createTestNetwork("null", "testnetwork", options.Generic{})
+	network, err := createTestNetwork("null", "testnull", options.Generic{})
```

Comment:
yeah, something seem wrong in the bridge driver as using the same id of previous existing networks triggers conflicts. Debugging...